examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,1181 @@
+{
```

Comment:
<p>(Only inference, we use pre-trained models) -&gt; in the intro you mention doing training</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>